warehouse/templates/email/environment-ignored-in-trusted-publisher/body.html

```diff
@@ -0,0 +1,59 @@
+{#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-#}
+{% extends "email/_base/body.html" %}
+
+{% set domain = request.registry.settings.get('warehouse.domain') %}
+
+{% block content %}
+  <p>
+    {% trans publisher_name=publisher.publisher_name, project_name=project_name, environment_name=environment_name,
```

Comment:
These emails shouldn't be marked as translated, we don't know the locale of the user that we're sending them to.

Reply:
fixed!